src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+const  Button = props => {
+  return (
+        <div>{props.children}</div>
+  )
+}

```

Comment:
- this component should accept an incoming `name` prop [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png)]

- `name` prop should be string (enforce this with `propTypes`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]

Reply:
It's advisable to not use the `React children` approach here........